journal/202103.md

```diff
@@ -0,0 +1,466 @@
+# {DRAFT} Decred Journal – March 2021
+
+![{alt text}](../img/{file.jpg} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+Highlights for March: {
+
+- consensus vote
+- v1.6.2 close
+- politeia mostly rewritten for a scalable and flexible storage backend
+- bison thing maybe
+
+}
+
+Contents: { sync at the end }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Integrations](#integrations)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Community Discussions](#community-discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+Merged in master and the upcoming v1.6.2 release:
+
+- only send [fast block](https://github.com/decred/dcrd/pull/2606) announcements to full node peers, and don't send them to lightweight clients that cannot properly handle them. This significantly improves dcrwallet's SPV operation. { how much better? }
```

Comment:
I was intentionally a little bit vague to avoid being super technical in the release notes, but it improves SPV connection stability and and, as a result, SPV wallets should need fewer rescans.

Reply:
It sounds too good to not mention it. If RNs don't have it, we get a bonus for exclusive content :P